temporal/api/cloud/auditlog/v1/message.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package temporal.api.cloud.auditlog.v1;
+
+option go_package = "go.temporal.io/api/cloud/auditlog/v1;auditlog";
+option java_package = "io.temporal.api.cloud.auditlog.v1";
+option java_multiple_files = true;
+option java_outer_classname = "MessageProto";
+option ruby_package = "Temporalio::Api::Cloud::AuditLog::V1";
+option csharp_namespace = "Temporalio.Api.Cloud.AuditLog.V1";
+
+import "google/protobuf/timestamp.proto";
+
+// LogRecord represents an audit log entry from Temporal, structured for easy parsing and analysis.
+message LogRecord {
+  // Time when the log was emitted from the source
+  google.protobuf.Timestamp emit_time = 1;
+
+  // Level of the log, i.e. info, warn, error etc
+  LogLevel level = 2;
+
+  // Operator email address
+  string user_email = 3;
+
+  // Operator IP address or server name
+  string caller_ip_address = 4;
+
+  // Operation performed
+  string operation = 5;
+
+  // Detailed information about the operation.
+  OperationDetail details = 6;
```

Comment:
Either call messages `OperationDetails` or field name `detail`. No need to mix singular and plural in this case IMO.

Reply:
I think we should keep the field name here, and maybe rename the message.  Note this format is already [published](https://docs.temporal.io/cloud/audit-logging#audit-log-format) for our customers to refer to.  When we change the fields we need to update the public facing document.  We can schedule an update when we release the endpoint and update the format after next week's v2 rollout.

Reply:
Changing the message name here is fine.

I do think this review should have the freedom to not have to match a published format for this beta cloud API. If it's just a doc update, no biggie IMO.